wayback/_client.py

```diff
@@ -419,6 +423,7 @@ def send(self, *args, **kwargs):
             if retries > 0:
                 seconds = self.backoff * 2 ** (retries - 1)
                 total_time += seconds
+                logger.debug("retrying after sleep of %s seconds", seconds)
```

Comment:
Minor nit: I think it would be helpful if this message were clearer that it is about to sleep, and hasn’t just woken up from sleeping, e.g:

```suggestion
                logger.debug("Will retry after sleeping for %s seconds...", seconds)
```

Or

```suggestion
                logger.debug("Sleeping for %s seconds before retrying...", seconds)
```